schema/extraction_config.schema.json

```diff
@@ -319,6 +319,25 @@
                     "description": "Another filter instance which is applied to the parent node"
                 }
             }
+        },
+        "rebrowse-triggers": {
+            "type": "object",
+            "unevaluatedProperties": false,
+            "description": "Subscribes server namespace nodes to rebrowse which can be filtered by namespaces.",
```

Comment:
This is a bit clunky, maybe "Configuration for triggering rebrowse based on changes to specific nodes".